### DISCUSSION TITLE
how to improve accuracy?

### DISCUSSION THREAD

**Discussion by behi123** [MilesCranmer/PySR#817] [Q&A]
I wrote this code to fit a symbolic regression model using PySRRegressor. However, the equations generated by the model do not achieve high accuracy. Here is the code I used:
X = data[["temp", "additive", "stress", "cycle"]].values
y = data["jnr"].values

scaler = StandardScaler()
X_scaled = scaler.fit_transform(X)

model = PySRRegressor(
    niterations=500,              
    binary_operators=["+", "-", "*", "/", "^"], 
    unary_operators=["exp", "log", "sqrt"],
    population_size=200,   
    populations=1000,
    maxsize=12,
    maxdepth=12,                 
    progress=True,                   
    loss="loss(x, y) = (x - y)^2",  
    model_selection="accuracy",         
    verbosity=1,                    
    random_state=42,                 
)

model.fit(X_scaled, y)
Despite trying to set reasonable values for parameters like maxsize, maxdepth, niterations, and using loss(x, y) = (x - y)^2, the accuracy of the generated equations is still low. How can I improve the performance of the model? Should I adjust the parameters or try a different approach? Any suggestions would be greatly appreciated!

**Comment by MilesCranmer** [comment]
Check out the tuning page: https://ai.damtp.cam.ac.uk/pysr/tuning/. Also worth browsing other discussions.
